relayer-cli/package.json

```diff
@@ -23,6 +24,10 @@
     "web3-batched-send": "^1.0.3"
   },
   "devDependencies": {
-    "ts-node": "^10.9.2"
+    "@types/jest": "^29.5.14",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "yarn": "^1.22.22"
```

Comment:
Confused about `yarn` defined again in devDependencies, and with a different version.

Reply:
@mani99brar I do not recall installing it explictly, so not sure why my local yarn decided to do so. I'll remove it. Thanks for the hint!